modeldb/modeldb-run.yaml

```diff
@@ -340,6 +340,9 @@
     - cat mosinit.hoc >> temp
     - mv temp  mosinit.hoc
 33975:
+    curate_patterns:
+    - pattern: 'hoc_run1: caught exception: hoc_execerror: We were unable to associate a PlayRecord item with a RANGE variable'
```

Comment:
I believe this is needed because the error is not printed in 8.2.2, not because it doesn't occur.